### PR TITLE
Adding konflux test tasks to pipeline

### PR DESCRIPTION
**Pull request by JoaoFula**
## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: .tekton/integration-tests/pipelines/lightspeed-integration-test-pipeline.yaml [hunk 1]
```diff
@@ -36,4 +36,30 @@ spec:
...
+      taskSpec:
+        params:
+          - name: SNAPSHOT
+          - name: SecretRef
+        steps:
+          - name: install-operator
+            env:
+              - name: SNAPSHOT
+                value: $(params.SNAPSHOT)
+            image: quay.io/redhat-user-workloads/crt-nshift-lightspeed-tenant/ols/bundle:on-pr-5690274f2c4ea8358eb73b942074c1dff684b723
```
Comment (tisnik) [inline .tekton/integration-tests/pipelines/lightspeed-integration-test-pipeline.yaml:60]: will it remain the same?

Comment (JoaoFula) [reply .tekton/integration-tests/pipelines/lightspeed-integration-test-pipeline.yaml:60]: no, for sure no. That's why I started the thread to see if we can trigger the bundle building in every PR
Then I could just have quay.io/redhat-user-workloads/crt-nshift-lightspeed-tenant/ols/bundle:on-pr-{{revision}}

**Review by tisnik — COMMENTED** [review]
it looks ok

**Pull request merged**